Bulls-and-cows/src/main/java/study/InputValidator.java

```diff
@@ -0,0 +1,12 @@
+package study;
+
+import static study.Constant.Integer.ONE;
+
+public class InputValidator {
+
+    public static void checkSize(int length) {
+        if ((length & ONE) != ONE) {
```

Comment:
```suggestion
        if (length != ONE) {
```
이렇게 하는게 더 직관적이지 않나요?